### PR TITLE
Feat api destinations crud

### PR DESCRIPTION
**Pull request by Adam0Schulz**

### CODE REVIEW

File: src/server/db/index.ts
Comment (bartbie) [inline src/server/db/index.ts:1]: Can't we just expose db object directly without wrapper functions?

File: src/server/routers/destinationsRouter.ts [hunk 1]
```diff
@@ -22,13 +22,13 @@ router.post("/", async (req, res) => {
 });
 
 //update
-router.put("/:id", (_req, _res) => {
-    // console.log("params", req.params);
-    // res.send(req.params)
+router.put("/:id", async (req, res) => {
+    res.status(200).json(
+        ok(await update(Destination, req.params.id, req.body)),
+    );
```
Comment (bartbie) [inline src/server/routers/destinationsRouter.ts]: there is no validation here, i assume mongoose will validate but it's not handled and will throw an error

**Comment by Adam0Schulz** [comment]
Oh and btw I can't run npm start and I think it's because the syntax of the underlying command is not compatible with windows terminal only linux and macos

**Comment by bartbie** [comment]
> Oh and btw I can't run npm start and I think it's because the syntax of the underlying command is not compatible with windows terminal only linux and macos

use git bash, i've used POSIX-compatible syntax

**Comment by bartbie** [comment]
> > Oh and btw I can't run npm start and I think it's because the syntax of the underlying command is not compatible with windows terminal only linux and macos
> 
> use git bash, i've used POSIX-compatible syntax

I will add cross-env or something similar to fix the problem later

**Review by bartbie — COMMENTED** [review]
lgtm, i've pushed my own changes regardless

**Review by matej-rolko — APPROVED** [review]

**Pull request merged**